buildconfig/Jenkins/Conda/build

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash -ex
+
+# This script expects a setup environment with all dependencies availiable in
+# the environment
+#
+# Expected args:
+#   1. WORKSPACE: path to the workspace/source code that this should run inside
+#   2. CMAKE_PRESET: the CMake preset that should be ran to generate the cmake files for this CI job
+#   3. ENABLE_DOCS: build the user docs
+#   4. ENABLE_DEV_DOCS: build the user docs
+#   5. ENABLE_BUILD_CODE: whether or not to build the main code target
+#   6. ENABLE_UNIT_TESTS: whether or not to build the unit tests target
+#   7. ENABLE_SYSTEM_TESTS: whether or not to build the system test data
+#   8. EXTRA_CMAKE_FLAGS: Extra flags to pass directly to cmake, enclose in "".
+
+WORKSPACE=$1
+CMAKE_PRESET=$2
+ENABLE_DOCS=$3
+ENABLE_DEV_DOCS=$4
+ENABLE_BUILD_CODE=$5
+ENABLE_UNIT_TESTS=$6
+ENABLE_SYSTEM_TESTS=$7
+EXTRA_CMAKE_FLAGS=$8
+
+# Only pass MANTID_DATA_STORE to CMake if present in the environment otherwise rely on CMake default
+if [![ -z "${MANTID_DATA_STORE}" ]]; then
+    MANTID_DATA_STORE_CMAKE="-DMANTID_DATA_STORE=${MANTID_DATA_STORE}"
+fi
+
+# Run CMake using preset and variables ${DIST_FLAGS} is from the environment
+cmake --preset=${CMAKE_PRESET} ${MANTID_DATA_STORE_CMAKE} ${DIST_FLAGS} ${EXTRA_CMAKE_FLAGS} $WORKSPACE
```

Comment:
That is my understanding, that ${DIST_FLAGS} is from the environment.

Reply:
@peterfpeterson Do you know where we use `DIST_FLAGS`? I can't find it set in Jenkins. Do you know where it is used or is it a relic we can remove?

Reply:
I thought it was related to the `CPACK_RPM_PACKAGE_RELEASE` which is actually set in [`buildscript`](https://github.com/mantidproject/mantid/blob/master/buildconfig/Jenkins/buildscript#L295-L300). It feels like it is [related to an old functionality](https://github.com/mantidproject/mantid/blob/master/buildconfig/CMake/CPackLinuxSetup.cmake#L47-L52) when we had just upgraded to cmake 3.6. 

It may be something that conda-build sets in the environment, but I was unable to find a link to confirm that.

I say delete the variable and see what happens.

Reply:
```suggestion
cmake --preset=${CMAKE_PRESET} ${MANTID_DATA_STORE_CMAKE} ${EXTRA_CMAKE_FLAGS} $WORKSPACE
```
Done!